File: src/main/java/DahRealPanda/plugins/colonyweb/colony/model/ItemCount.java
package DahRealPanda.plugins.colonyweb.colony.model;

/**
 * An item with a quantity — used for citizen inventories and research costs.
 */
public class ItemCount extends ItemInfo {
    public int count;
    public int slot = -1; // inventory slot, -1 when not slot-bound
}

File: common/src/main/kotlin/DahRealPanda/plugins/colonyweb/model/ItemCount.kt
package DahRealPanda.plugins.colonyweb.model

/**
 * An item with a quantity — used for citizen inventories and research costs.
 */
data class ItemCount(
    @JvmField var count: Int = 0,
    @JvmField var slot: Int = -1 // inventory slot, -1 when not slot-bound
) : ItemInfo()
